docs-gen/README.md

```diff
@@ -18,7 +14,7 @@ The static page is generated with:
 - [Learn Theme](https://themes.gohugo.io/hugo-theme-learn/)
 
 Please follow the [documentation](https://gohugo.io/documentation/) for
-installation and further questions around the framework.
+installation and further questions around the framework. Currently, the minimal tested HUGO version vor VSS is `0.78.1`
```

Comment:
spelling of for

Reply:
Fixed. You are good at spotting these 😁

Reply:
You are welcome but I should warn that once I find one or two I tend to stop looking so closely and leave spell checking the rest to the author. That is the case here.. :)